packages/client/hmi-client/src/api/api.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import useAuthStore from '../stores/auth';
+
+const API = axios.create({
```

Comment:
are all endpoints going to be `/api`? if not is the expectations that we will have multiple axios instances present?

Reply:
I think as far as the client is concerned, it will use the `/api` prefix. We can have debugging/administrative routes with other names that are out of scope of the client application.